src/seq.rs

```diff
@@ -1119,6 +1240,111 @@ impl QueueTempo {
     pub fn set_skew_base(&self, value: u32) { unsafe { alsa::snd_seq_queue_tempo_set_skew_base(self.0, value as c_uint) } }
 }
 
+/// [snd_seq_queue_status_t](http://www.alsa-project.org/alsa-doc/alsa-lib/group___seq_queue.html) wrapper
+pub struct QueueStatus(*mut alsa::snd_seq_queue_status_t);
+
+unsafe impl Send for QueueStatus {}
+
+impl Drop for QueueStatus {
+    fn drop(&mut self) { unsafe { alsa::snd_seq_queue_status_free(self.0) } }
+}
+
+impl QueueStatus {
+    fn new() -> Result<Self> {
+        let mut q = ptr::null_mut();
+        acheck!(snd_seq_queue_status_malloc(&mut q)).map(|_| QueueStatus(q))
+    }
+
+    /// Creates a new QueueStatus with all fields set to zero.
+    pub fn empty() -> Result<Self> {
+        let q = QueueStatus::new()?;
+        unsafe { ptr::write_bytes(q.0 as *mut u8, 0, alsa::snd_seq_queue_status_sizeof()) };
+        Ok(q)
+    }
+
+    pub fn get_queue(&self) -> i32 { unsafe { alsa::snd_seq_queue_status_get_queue(self.0) as i32 } }
+    pub fn get_events(&self) -> i32 { unsafe { alsa::snd_seq_queue_status_get_events(self.0) as i32 } }
+    pub fn get_tick_time(&self) -> u32 { unsafe {alsa::snd_seq_queue_status_get_tick_time(self.0) as u32 } }
+    pub fn get_real_time(&self) -> time::Duration { unsafe {
+        let t = &(*alsa::snd_seq_queue_status_get_real_time(self.0));
+        time::Duration::new(t.tv_sec as u64, t.tv_nsec as u32)
+    } }
+    pub fn get_status(&self) -> u32 { unsafe { alsa::snd_seq_queue_status_get_status(self.0) as u32 } }
+}
+
+/// [snd_seq_remove_events_t](https://www.alsa-project.org/alsa-doc/alsa-lib/group___seq_event.html) wrapper
+pub struct RemoveEventsInfo(*mut alsa::snd_seq_remove_events_t);
+
+unsafe impl Send for RemoveEventsInfo {}
+
+impl Drop for RemoveEventsInfo {
+    fn drop(&mut self) { unsafe { alsa::snd_seq_remove_events_free(self.0) } }
+}
+
+impl RemoveEventsInfo {
+    pub fn new() -> Result<Self> {
+        let mut q = ptr::null_mut();
+        acheck!(snd_seq_remove_events_malloc(&mut q)).map(|_| RemoveEventsInfo(q))
+    }
+
+
+    pub fn get_condition(&self) -> RemoveEventsCondition { unsafe { 
+        RemoveEventsCondition::from_bits_truncate(alsa::snd_seq_remove_events_get_condition(self.0) as u32) 
+    } }
+    pub fn get_queue(&self) -> i32 { unsafe { alsa::snd_seq_remove_events_get_queue(self.0) as i32 } }
+    pub fn get_time(&self) -> time::Duration { unsafe {
+        let mut d = alsa::snd_seq_timestamp_t { data: (*alsa::snd_seq_remove_events_get_time(self.0)).data };
+        let t = &(*d.time());
+        
+        time::Duration::new(t.tv_sec as u64, t.tv_nsec as u32)
+    } }
+    pub fn get_tick(&self) -> u32 { unsafe {
+        let mut d = alsa::snd_seq_timestamp_t { data: (*alsa::snd_seq_remove_events_get_time(self.0)).data };
+        
```

Comment:
`alsa::snd_seq_timestamp_t` is a union so it contains either a tick or a time, or both. 
Can you double check that both `get_time` and `get_tick` be used the way you do here?

Reply:
Oups, my bad. good catch.   Definitely used as time here, not tick. Removed the tick getters/setters.